lib/sequent/core/event_store.rb

```diff
@@ -51,7 +51,34 @@ def commit_events(command, streams_with_events)
       end
 
       ##
-      # Returns all events for the aggregate ordered by sequence_number
+      # Returns all events for the aggregate ordered by sequence_number, disregarding snapshot events
+      #
+      def stream_events_for_aggregate(aggregate_id, load_until: nil)
+        stream = get_event_stream(aggregate_id)
+        fail ArgumentError, 'no stream found for this aggregate' if stream.blank?
+
+        q = Sequent
+          .configuration
+          .event_record_class
+          .where(aggregate_id: aggregate_id)
+          .where.not(event_type: Sequent.configuration.snapshot_event_class.name)
+          .order(:sequence_number)
+        q = q.where('created_at < ?', load_until) if load_until.present?
+        fail ArgumentError, 'no events for this aggregate' if q.blank?
```

Comment:
If there are lots of events this will do a possible expensive count query first. Perhaps we can track this in the `in_batches` loop to see if there were any events at all.